tests/dbus/test-client

```diff
@@ -99,7 +88,22 @@ send "commissioner joiner add * ABCDEF\r\n"
 expect "Done"
 wait
 EOF
-    sleep 15
+    sleep 5
+    expect <<EOF &
```

Comment:
nit: better to add comment to explain why introducing the `mtd 2` here for readability.